src/pages/das-api/guides/get-collection-nfts.md

```
@@ -0,0 +1,284 @@
+---
+title: Get All Tokens in a Collection
+metaTitle: Get All Tokens in a Collection | DAS API Guides
+description: Learn how to retrieve all digital assets belonging to a specific collection
+---
+
+# Get All Tokens in a Collection
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)